cmd/auth_e2e_test.go

```diff
@@ -0,0 +1,262 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/kr/pty"
+	"github.com/stretchr/testify/require"
+	"github.com/xata/cli/internal/envcfg"
+)
+
+type TestConfig struct {
+	TestKey        string `env:"TEST_API_KEY"`
+	TestBinaryPath string `env:"TEST_XATA_PATH"`
+}
+
+const defaultXataCommand = "../xata"
+
+func GetTestConfigFromEnv() (config TestConfig, err error) {
+	err = envcfg.ReadEnv([]string{"../.env", "../.env.local"})
+	if err != nil {
+		return TestConfig{}, err
+	}
+
+	config.TestKey = os.Getenv("TEST_API_KEY")
+	config.TestBinaryPath = os.Getenv("TEST_XATA_PATH")
+	if config.TestBinaryPath == "" {
+		config.TestBinaryPath = defaultXataCommand
+	}
+	config.TestBinaryPath, err = filepath.Abs(config.TestBinaryPath)
+	if err != nil {
+		return TestConfig{}, err
+	}
+	return
+}
+
+// NewVT10XConsole returns a new expect.Console that multiplexes the
+// Stdin/Stdout to a VT10X terminal, allowing Console to interact with an
+// application sending ANSI escape sequences.
+func NewVT10XConsole(opts ...expect.ConsoleOpt) (*expect.Console, error) {
+	ptm, pts, err := pty.Open()
+	if err != nil {
+		return nil, err
+	}
+
+	term := vt10x.New(vt10x.WithWriter(pts))
+
+	c, err := expect.NewConsole(append(opts, expect.WithStdin(ptm), expect.WithStdout(term), expect.WithCloser(pts, ptm))...)
+	if err != nil {
+		return nil, err
+	}
+
+	return c, nil
+}
+
+func TestAuthLoginCommand(t *testing.T) {
+	config, err := GetTestConfigFromEnv()
+	require.NoError(t, err)
+	require.NotEmpty(t, config.TestKey)
```

Comment:
This will make the error more user friendly (this is what's failing in CI):

```suggestion
	require.NotEmpty(t, config.TestKey, "env variable 'TEST_API_KEY' must be present to test")
```